src/test/java/io/jenkins/plugins/oidc_provider/IdTokenFileCredentialsTest.java

```diff
@@ -74,7 +73,8 @@ public class IdTokenFileCredentialsTest {
         assertEquals(r.jenkins.getRootUrl() + "oidc", claims.getIssuer());
     }
 
-    @Test public void declarative() throws Exception {
```

Comment:
Also I notice that there is no visual separation between test cases now, which is a major regression when it comes to picking out log messages from various test methods in a class.

```
…
[WARNING] [stderr]    5.381 [id=57]	INFO	o.j.h.t.TemporaryDirectoryAllocator#dispose: deleting …/oidc-provider-plugin/target/tmp/j h13148824964621155285
[INFO] [stdout] === Starting smokes(io.jenkins.plugins.oidc_provider.IdTokenFileCredentialsTest)
[WARNING] [stderr]    0.063 [id=194]	INFO	o.jvnet.hudson.test.JenkinsRule#createWebServer2: Running on http://localhost:41395/jenkins/
…
```